docs/data-apps/attribution-modeling/index.md

```diff
@@ -41,46 +42,55 @@ Use the toggle `Last N days View (Dynamic)` to choose whether your would like to
 
 **Defining a Last N Days (Dynamic) View**
 
-The so-called `Dynamic` views are to be used for generating datasets that have a rolling conversion window of last nth day and will be refreshed automatically (e.g. Last 30 days). The app will save the last-refreshed date with the View configurations and any subsequent day a user logs back in the app, a query will run in the background to look for any newly processed conversion event in the conversion source and if there is, the dynamic datasets are refreshed by running all the queries that are needed to generate data for the charts to populate. Once the update finishes the conversion window should display the new date range.
+The so-called `Dynamic` views are to be used for generating datasets that have a rolling conversion window of last nth day and will be refreshed automatically (e.g. Last 30 days). The app will save the last-refreshed date with the View configurations and any subsequent day a user logs back in the app, a query will run in the background to look for any newly processed conversion event in the conversion source and if there is, the dynamic datasets are refreshed by running all the queries that are needed to generate data for the charts to populate. 
 
-If you choose this option, set the `auto-update days`: the number of days since the last conversion event defined here will define the conversion window.
+If you choose this option, set the `auto-update days`: the number of days since the last conversion event defined here will define the conversion window. The latest conversion window in use can be checked on the `Settings` page where a table with information on all the created views is displayed including conversion window that is currently in use.
 
 **Defining a Custom Date Range (static) View**
 
 Non-dynamic views will have to be given a name and will typically be used to generate a fixed dataset (e.g. Jan, Q1, 2023) to avoid having to recalculate the analysis for subsequent users.
 
 Define a fixed conversion window by selecting the appropriate date range with the date picker tool (which gets activated by clicking on the default date range). 
 
-#### 1.2 Set a `currency symbol` (defaults to £)
-
-#### 1.3 Decide if you would like to `use Non-Conversions`:
-
-Use this with caution, currently it uses the `snowplow_attribution_paths_to_non_conversion` table as is without considering the conversion period. The intention is to make this a fully automated feature in the near future so watch for updates on this.
-
+#### 1.2 Set a currency symbol (defaults to £)
```

Comment:
I think we should change this to be $ by default - we are now officially a US company

Reply:
I added it to a new PR on the app so this can be changed here as well now.